code/modules/admin/verbs/cinematic.dm

```diff
@@ -1,21 +1,16 @@
-/client/proc/cinematic(cinematic as anything in list("explosion", null))
-	set name = "cinematic"
-	set category = "Debug"
-	set desc = "Shows a cinematic."	// Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
-	set hidden = 1
+/client/proc/cinematic()
+	set name = "Cinematic"
+	set category = "Event"
+	set desc = "Shows a cinematic." // Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
+	set hidden = TRUE
+
 	if(!SSticker)
 		return
+	if(!check_rights(R_EVENT))
```

Comment:
Just noticed that this is actually still defined in `admin_verbs_maintainer`, which implies that it could be kind of gamebreaking/could fuck up people's screens.

You should probably keep it under maintainer tbh.
```
// In this list are verbs that should ONLY be executed by maintainers, aka people who know how badly this will break the server
// If you are adding something here, you MUST justify it
GLOBAL_LIST_INIT(admin_verbs_maintainer, list(
	/client/proc/ticklag, // This adjusts the server tick rate and is VERY easy to hard lockup the server with
	/client/proc/debugNatureMapGenerator, // This lags like hell, and is very easy to nuke half the server with
	/client/proc/vv_by_ref, // This allows you to lookup **ANYTHING** in the server memory by spamming refs. Locked for security.
	/client/proc/cinematic, // This will break everyone's screens in the round. Dont use this for adminbus.
	/client/proc/throw_runtime, // Do I even need to explain why this is locked?
))
```